www/templates/layout.html

```diff
@@ -6,6 +6,8 @@
     {% block head %}
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='http://fonts.googleapis.com/css?family=Pacifico&amp;subset=latin' rel='stylesheet' type='text/css'>
```

Comment:
Maybe we can grab the full font and add it via the production files? So we have it all locally hosted